### ISSUE TITLE
Wrong doc ?

### ISSUE THREAD

**Issue by gwenth** [esbenp/bruno#9]
Hi ! 

Thanks for this package, which is really useful :)
I just wanted to point out the fact that the doc seems pretty old: It says 

> composer require optimus/bruno ~1.0

when the latest release seems to be 2.0.5 (and 1.0 is not working with latest laravel version: 5.4)

Similarly, it says to call 

> use Optimus\Api\Controller\EloquentBuilderTrait;
> use Optimus\Api\Controller\LaravelController;

when the namespace seems to have changed in favor of Optimus\Bruno.

That's it :) 
Maybe I'm wrong and this point is not relevant, but just in case...

Have a nice day and thanks again !

**Comment by justinwilson-apollidon** [comment]
Yeah this package seems abandoned for the most part. I wouldn't expect the docs to be corrected at this point. Docs still haven't been updated since you created this issue in March.

**Comment by muhalovic** [comment]
Yup. The docs still haven't been updated yet.
